plugins/base/macro-condition-filter.cpp

```diff
@@ -142,6 +147,45 @@ std::string MacroConditionFilter::GetShortDesc() const
 	return "";
 }
 
+void MacroConditionFilter::SetCondition(Condition cond)
+{
+	_condition = cond;
+	SetupTempVars();
+}
+
+void MacroConditionFilter::SetupTempVars()
+{
+	MacroCondition::SetupTempVars();
+	switch (_condition) {
+	case Condition::ENABLED:
+		break;
+	case Condition::DISABLED:
+		break;
+	case Condition::SETTINGS_MATCH:
+		AddTempvar("settings",
+			   obs_module_text(
+				   "AdvSceneSwitcher.tempVar.filter.settings"));
+		break;
+	case Condition::SETTINGS_CHANGED:
+		AddTempvar("settings",
```

Comment:
I think a good one here and in similar source one would be something like `changedSettings` with only changed ones

Reply:
Thanks for the suggestion! :)

While I agree that it might be nice to have I can't really think of good use case for now, since the option already exists to watch for changes of individual settings.
So for now I will add it to the todo list, but I am not sure when I will get around to it.

Reply:
Use case is for example debugging other extensions or checking what changes so that you can set up said individual settings properly. Not a big priority, but something nice to have IMO